### PR TITLE
Add #[no_std] support

### PR DESCRIPTION
**Pull request by cedric-h**
wooooo

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -10,20 +10,22 @@ readme = "README.md"
 keywords = ["ecs", "entity"]
 categories = ["data-structures", "game-engines"]
 
-[badges]
-maintenance = { status = "actively-developed" }
-
```
Comment (Ralith) [inline Cargo.toml]: Why was this moved?

File: Cargo.toml [hunk 2]
```diff
@@ -10,20 +10,22 @@ readme = "README.md"
...
 [features]
+default = ["std"]
+std = []
 # Enables derive(Bundle) and derive(Query)
 macros = ["hecs-macros", "once_cell"]
 
 [dependencies]
-fxhash = "0.2.1"
 hecs-macros = { path = "macros", version = "0.1.0", optional = true }
 once_cell = { version = "1.2", optional = true }
```
Comment (Ralith) [inline Cargo.toml]: This isn't no-std compatible.

File: Cargo.toml [hunk 3]
```diff
@@ -10,20 +10,22 @@ readme = "README.md"
 keywords = ["ecs", "entity"]
 categories = ["data-structures", "game-engines"]
```
Comment (Ralith) [inline Cargo.toml]: Let's add `no-std` here to celebrate.

File: Cargo.toml (hunk 1 above)
Comment (cedric-h) [reply Cargo.toml]: I was lazy and used `cargo add hashbrown`, and somewhere in the reading/writing process it decided to also move this.
